Mathlib/Analysis/Normed/Field/Lemmas.lean

```diff
@@ -291,6 +291,26 @@ end NormedDivisionRing
 
 namespace NormedField
 
+lemma discreteTopology_or_nontriviallyNormedField (𝕜 : Type*) [h : NormedField 𝕜] :
```

Comment:
Given that both in the statement and in the proof you use some non hyper-standard tools (the way you state non-triviality, and the `config` setting), adding a bit of doc would be very useful.